packages/gestalt/src/Checkbox.js

```diff
@@ -100,7 +102,7 @@ export default function Checkbox({
         marginRight={-1}
       >
         <Label htmlFor={id}>
-          <Box paddingX={1} position="relative">
+          <Box ref={forwardedRef} paddingX={1} position="relative">
             <input
```

Comment:
@AlbertCarreras could we set the ref on the `input type="checkbox"` instead. That way people can set focus on the element if they want to.

Reply:
DONE